Havit.Blazor.Components.Web.Bootstrap/Toasts/HxMessenger.razor.cs

```diff
@@ -11,6 +11,11 @@ public partial class HxMessenger : ComponentBase, IDisposable
 		/// </summary>
 		[Parameter] public ToastContainerPosition Position { get; set; } = ToastContainerPosition.None;
 
+		/// <summary>
+		/// Fires when a message (a toast) is hidden by close-button click or through auto-hide.
+		/// </summary>
+		[Parameter] public EventCallback<MessengerMessage> OnMessageClosed { get; set; }
```

Comment:
@Harvey1214 The `HxMessenger` is usualy sitting in `App.razor` or `MainLayout.razor`. I don't think you would want to listen to this callback there as the messages are added from the user-code via `IHxMessengerService` and that is the place where you "know" what you want to do when the message is "closed".

This just adds my doubts whether this whole feature is a good idea.